web/sites/default/config/views.view.council_documents.yml

```diff
@@ -1247,6 +1527,44 @@ display:
           expose:
             operator_limit_selection: false
             operator_list: {  }
+        status:
```

Comment:
The taxonomy term publish filter was added to the "confirmed date and time" block which doesn't have any taxonomy term relationship, assuming that was meant to go on the "assigned documents" block instead?

![image](https://github.com/user-attachments/assets/e1076ee5-3628-4370-b1bf-16651f70719b)